src/debug.rs

```diff
@@ -1,10 +1,37 @@
 /// The native debug interface exposed to the ewasm contract. These functions are for testing
 /// purposes only. On a live VM, any bytecode trying to import these symbols will be rejected.
-extern "C" {
-    pub fn debug_print32(value: u32);
-    pub fn debug_print64(value: u64);
-    pub fn debug_printMem(offset: *const u32, len: u32);
-    pub fn debug_printMemHex(offset: *const u32, len: u32);
-    pub fn debug_printStorage(pathOffset: *const u32);
-    pub fn debug_printStorageHex(pathOffset: *const u32);
+
```

Comment:
Nitpicking, but I prefer if all attributes (documentation comments are syntactic sugar for `#[doc]`) have no whitespace between them and the items they apply to.

Reply:
But they don't apply to `mod native`, but the entire file.

Reply:
In that case, you need to prefix with `//!`. otherwise the attribute will be applied to the item below rather than the surrounding module.

Reply:
Done.